test/Test/Pos/BinarySpec.hs

```diff
@@ -58,40 +58,42 @@ tinyVarIntSpec = describe "TinyVarInt" $ do
                 B.biSize (B.TinyVarInt n) `shouldBe` b
         maxnum = 2^(14::Int)-1
 
-    describe "1 byte" $ do
```

Comment:
You will see how I have marked some specs below as `pending`. The reason is that I would like to clarify if they are now redundant or not. In particular, due to the rework of a `TinyVarInt`, we can now encode things bigger than `2^14-1`. Should we forbid this? Should we allow this?